### PR TITLE
Intake astro2

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

File: recipes/intake-astro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - jinja2
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/intake-astro/meta.yaml]: Do you need `python >=3.5` only to build but it runs on `2.7`? If not you need to use `>=3.5` here too.

File: recipes/intake-astro/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.1" %}
+{% set name = "intake-astro" %}
+{% set sha256 = "8f2a5f49a10a52cd58770154075de27229242e9ae5c4e95c6a449b285653b90d" %}
+
+package:
+   name: {{ name }}
+   version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/intake-astro/meta.yaml]: This line is not really needed.

File: recipes/intake-astro/meta.yaml (hunk 1 above)
Comment (martindurant) [reply recipes/intake-astro/meta.yaml]: For the moment, Intake itself only runs on python 3.5+, so it's implicit, but that may change. There is nothing non-py2 in this repo, so I'm not sure.

Comment (ocefpaf) [reply recipes/intake-astro/meta.yaml]: In that case I would constrain the plugin to `>=3.5` too. I'm not sure what are the side effects of letting the plugin be installable on py2.7 when a dependency is not: unsatisfied deps or Python upgrade? One is OK the other may surprise users.

**Comment by martindurant** [comment]
OK, I think it's all good except for the one azure run, whic I've seen fail like that elsewhere (i.e., not connected to the PR as such)

File: recipes/intake-astro/meta.yaml (hunk 1 above)
Comment (martindurant) [reply recipes/intake-astro/meta.yaml]: done

**Comment by martindurant** [comment]
Thank you, and sorry for the big ball of trouble I caused here.

**Comment by ocefpaf** [comment]
> Thank you, and sorry for the big ball of trouble I caused here.

No problem. IMO we should improve the style standards and linter to help us with that, not your fault.

**Pull request merged**